### PR TITLE
Working on automating build releases for merges to the main

### LINKED ISSUE #168: Automated Release for the Builds
Given we are using github,
when a build is done then there should be an automated Release for the software.

### PR DESCRIPTION
**Pull request by kpothuganti**
…releases whenever a merge to the main branch happens

Fixes #168 

**What was changed?**

*Added a release.yml file to the workflows folder in order to automate the creation of build releases with a merge to the main branch*

**Why was it changed?**

*Currently, there aren't any build releases being made when a merge to the main branch occurs. This .yml file is meant to help automate this process with each merge to the main.*

**How was it changed?**

*I had to add a new file to the workflows folder in order to automate the process of build release creation. I had to include sections to ensure that this is compatible for both mac and windows as well as the python version required.*

**Screenshots that show the changes (if applicable):**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: Build, Test, and Release on macOS and Windows
+
+on:
+  push:
+    branches:
+      - main  # Trigger on merge to the main branch
```
Comment (emrameduseljac) [inline .github/workflows/release.yml:6]: Comment unnecessary, self-explanatory line of code.

**Review by emrameduseljac — COMMENTED** [review]
Unable to test code before it is merged to main - only critique are the unnecessary comments.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (kate-holdener) [inline .github/workflows/release.yml:46]: Do you have the necessary tokens set up in GitHub secrets?

File: .github/workflows/release.yml (hunk 1 above)
Comment (kate-holdener) [inline .github/workflows/release.yml:6]: You should temporarily add the current branch to this list for testing. Where will these releases go?

**Pull request merged**